ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousPlanner.java

```diff
@@ -180,6 +179,22 @@ public FootstepPlan generateAStarFootstepPlan(HeightMapData heightMapData,
       request.setSnapGoalSteps(true);
       request.setAbortIfGoalStepSnappingFails(true);
 
+      // When walking backwards, we want to keep the body facing in the same direction, otherwise the robot will turn as it tries to step backwards
+      if (command.getWalkBackwards())
```

Comment:
This is where we want to make sure we face the same way when we start walking backwards